addons/event/views/event_views.xml

```diff
@@ -340,51 +340,45 @@
                     <field name="legend_blocked"/>
                     <field name="legend_normal"/>
                     <field name="legend_done"/>
-                    <field name="duration_str"/>
                     <field name="activity_ids"/>
                     <field name="activity_state"/>
                     <templates>
                         <t t-name="kanban-box">
                             <div t-attf-class="{{!selection_mode ? 'oe_kanban_color_' + kanban_getcolor(record.color.raw_value) : ''}} oe_kanban_card oe_kanban_global_click">
-                                <div class="o_event_left text-center">
+                                <div class="o_event_left text-center d-flex flex-column justify-content-center">
                                     <div class="o_day"><t t-esc="record.date_begin.raw_value.getDate()"/></div>
                                     <div>
                                         <t t-esc="moment(record.date_begin.raw_value).format('MMM')"/>
                                         <t t-esc="record.date_begin.raw_value.getFullYear()"/>
                                     </div>
                                     <div><t t-esc="moment(record.date_begin.raw_value).format('LT')"/></div>
+                                    <div t-if="moment(record.date_begin.raw_value).format('D MMM') !== moment(record.date_end.raw_value).format('D MMM')">
```

Comment:
You completely ignored my comment that said:

> Would be cleaner to compare the `.getDayOfYear()` instead of comparing strings.

It makes me very sad.

Reply:
ahaha sorry x'D